src/app/setup.cfg

```diff
@@ -10,3 +10,6 @@ exclude = test/api
 [tool:pytest]
 xfail_strict=true
 
+[isort]
```

Comment:
This is why there's a bunch of import re-ordering in the diffs.

Reply:
Yeah, this should be done for *both* beer_garden and brewtils, IMO.